### PR TITLE
[Feature] Support alter Iceberg resource and desc all Iceberg table

### PR DESCRIPTION
**Pull request by mxdzs0612**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

- support `alter resource <iceberg_resource>`  command
- support `describe <iceberg_table> all` command

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 3 / 20 (15.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/IcebergResource.java|0|16|00.00%|
|:large_blue_circle:|com/starrocks/catalog/ResourceMgr.java|2|3|66.67%|
|:large_blue_circle:|com/starrocks/analysis/DescribeStmt.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java [hunk 1]
```diff
@@ -111,4 +113,34 @@ public String getIcebergImpl() {
...
     }
+
+    /**
+     * <p>alter the resource properties.</p>
+     * <p>the user can not alter the property that the system does not support.
+     *
+     * @param properties the properties that user uses to alter
+     * @throws DdlException
+     */
+    public void alterProperties(Map<String, String> properties) throws DdlException {
```
Comment (DorianZheng) [inline fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java:124]: Plz add ut

**Comment by mxdzs0612** [comment]
run starrocks_fe_unittest

**Comment by mxdzs0612** [comment]
run starrocks_fe_unittest

**Comment by mxdzs0612** [comment]
run starrocks_fe_unittest

**Comment by mxdzs0612** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 13 / 20 (65.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/IcebergResource.java|10|16|62.50%|
|:large_blue_circle:|com/starrocks/catalog/ResourceMgr.java|2|3|66.67%|
|:large_blue_circle:|com/starrocks/analysis/DescribeStmt.java|1|1|100.00%|

**Comment by mxdzs0612** [comment]
@DorianZheng @caneGuy PTAL

**Review by caneGuy — APPROVED** [review]

**Review by DorianZheng — APPROVED** [review]
LGTM

**Review by dirtysalt — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java [hunk 2]
```diff
@@ -111,4 +113,34 @@ public String getIcebergImpl() {
...
+            String key = entry.getKey();
+            String value = entry.getValue();
+            if (ICEBERG_METASTORE_URIS.equals(key)) {
+                if (StringUtils.isBlank(value)) {
+                    throw new DdlException(ICEBERG_METASTORE_URIS + " can not be null");
+                }
+                validateMetastoreUris(value);
+                this.metastoreURIs = value;
+            } else if (ICEBERG_IMPL.equals(key)) {
+                if (StringUtils.isBlank(value)) {
```
[the remark this answers is not in the document]
Comment (mxdzs0612) [reply fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java:137]: Well, if custom catalog class has been loaded into FE lib, users don't need to restart the FE server. Custom catalog only takes effects when creating tables.

**Pull request merged**